src/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Typography from "@mui/material/Typography";
+import CardMedia from "@mui/material/CardMedia";
+
+import { Box, StyledCard, CardContent } from "./styled";
+
+const Profile = ({ data, isSuccess }) => {
+  return (
+    <Box>
+      {isSuccess &&
```

Comment:
isSuccess is not necessary. You can use:
profileData && <div>{profileData.somethingLikeNameOrAge}</div>